src/main/java/cloud/orbit/spring/actuate/OrbitHealthIndicator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ Copyright (C) 2017 Electronic Arts Inc.  All rights reserved.
+
+ Redistribution and use in source and binary forms, with or without
+ modification, are permitted provided that the following conditions
+ are met:
+
+ 1.  Redistributions of source code must retain the above copyright
+     notice, this list of conditions and the following disclaimer.
+ 2.  Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in the
+     documentation and/or other materials provided with the distribution.
+ 3.  Neither the name of Electronic Arts, Inc. ("EA") nor the names of
+     its contributors may be used to endorse or promote products derived
+     from this software without specific prior written permission.
+
+ THIS SOFTWARE IS PROVIDED BY ELECTRONIC ARTS AND ITS CONTRIBUTORS "AS IS" AND ANY
+ EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ DISCLAIMED. IN NO EVENT SHALL ELECTRONIC ARTS OR ITS CONTRIBUTORS BE LIABLE FOR ANY
+ DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package cloud.orbit.spring.actuate;
+
+import org.springframework.boot.actuate.health.*;
+
+import cloud.orbit.actors.Stage;
+import cloud.orbit.actors.runtime.NodeCapabilities;
+
+public class OrbitHealthIndicator implements HealthIndicator
+{
+    private final Stage stage;
+
+    @SuppressWarnings("WeakerAccess")
+    public OrbitHealthIndicator(final Stage stage)
+    {
+        this.stage = stage;
+    }
+
+    @Override
+    public Health health()
+    {
+        if (stage == null)
```

Comment:
I think this class should just "work" without any extra configuration on the part of the user. Especially since this is an auto-configured component that can be disabled or overridden, if the user wants anything specific they should be able to override it like they would with any auto-configured component.

Notice the
```java
@ConditionalOnEnabledHealthIndicator("orbit")
```
and
```java
@ConditionalOnMissingBean(name = "orbitHealthIndicator")
```
annotations in the `OrbitHealthIndicatorConfiguration` class.